spec/registry/gcpDataproc/facets/GcpDataprocRunFacet.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/1-0-0/GcpDataprocRunFacet.json",
+  "$defs": {
+    "GcpDataprocRunFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/RunFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "appId": {
+              "description": "Spark application id",
```

Comment:
Application ID set in the spark configuration of the current context. Its format depends on the resource manager.